LSPS1/README.md

```diff
@@ -288,71 +280,77 @@ The client MAY check the current status of the order at any point.
-
-> **Rationale** On-chain payments are required for payments with higher amounts, especially to push `client_balance_sat` to the client. On-chain payments are also useful to onboard new users to Lightining. On the other hand, Lightning payments are the preferred way to do payments because they are quick and easily refundable.
-
-
-**Payment object**
+This section describes the `payment` object returned by `lsps1.create_order` and `lsps1.get_order`.
 
 ```json
 {
+  "bolt11": {
+    "state" : "EXPECT_PAYMENT",
+    "expires_at": "2025-01-01T00:00:00Z",
+    "fee_total_sat": 8888,
+    "order_total_sat": "200888",
+    "bolt11_invoice": "lnbc252u1p3aht9ysp580g4633gd2x9lc5al0wd8wx0mpn97..."
+  },
+  "onchain": {
     "state": "EXPECT_PAYMENT",
-    "fee_total_sat": "8888",
-    "order_total_sat": "2008888",
-    "bolt11_invoice": "lnbc252u1p3aht9ysp580g4633gd2x9lc5al0wd8wx0mpn97...",
+    "expires_at": "2025-01-01T00:00:00Z",
+    "fee_total_sat": 9999,
+    "order_total_sat": 200999,
     "onchain_address": "bc1p5uvtaxzkjwvey2tfy49k5vtqfpjmrgm09cvs88ezyy8h2zv7jhas9tu4yr",
     "min_onchain_payment_confirmations": 1,
-    "min_fee_for_0conf": 253,
-    "onchain_payment": {
-        "outpoint": "0301e0480b374b32851a9462db29dc19fe830a7f7d7a88b81612b9d42099c0ae:1",
-        "sat": "1200",
-        "confirmed": false
+    "min_fee_for_0conf": 253
     }
-},
+  }
+}
 ```
 
-- `state <string enum>` MUST be one of these values:
+The Client MUST ignore unkown payment options.
+
+The LSP MAY omit payment options.
+
+> **Rationale**: E.g.: The LSP might not support onchain payments.
+
+> **Rationale**:
+> Fees and expiry dates are defined on the level of the payment-option.
+> This allows an LSP to provide the cheapest service for each payment option.
+>
+> An LSP might choose to have a lower fee for lightning than for onchain payments.
+> Onchain payments might
+> - be more costly for the LSP as they might have to spend a small UTXO in the future
+> - be more risky for the LSP in case of 0-conf channels
+> - require longer confirmation times than a lightning payment. (Risk of changing on-chain fees)
+
+
+#### 3.1 Lightning Payments using BOLT-11
+
+```json
+{
+    "state" : "EXPECT_PAYMENT",
+    "expires_at": "2025-01-01T00:00:00Z",
+    "fee_total_sat": 8888,
+    "order_total_sat": "200888",
+    "bolt11_invoice": "lnbc252u1p3aht9ysp580g4633gd2x9lc5al0wd8wx0mpn97..."
+}
+```
+
+- `state`
     - `EXPECT_PAYMENT` Payment expected.
     - `HOLD` Lighting payment arrived, preimage NOT released.
-    - `PAID` Lightning payment arrived, preimage released OR full `order_total_sat` on-chain payment arrived.
-    - `REFUNDED` Lightning payment or on-chain payment has been refunded.
+    - `PAID`  When the has been preimage released
+    - `REFUNDED` Lightning payment has been refunded.
```

Comment:
Turn `REFUNDED` into `CANCELED`. If no payment arrived within the payment expiry time, setting it to `REFUNDED` would look weird.

I think we had this discussion before.